src/seed.go

```diff
@@ -108,9 +111,13 @@ func seed(path string, database *data.Database) {
 		aesKeys[i].Metadata.CreationDate = time.Now().Unix()
 		aesKeys[i].Metadata.Enabled = true
 		aesKeys[i].Metadata.KeyManager = "CUSTOMER"
-		aesKeys[i].Metadata.KeyState = "Enabled"
```

Comment:
I think we still want the default `KeyState` to be Enabled here. And then we override it below if it needs setting to Pending.

So `aesKeys[i].Metadata.KeyState = cmk.KeyStateEnabled`

Or is there a reason we shouldn't be setting it?

Reply:
Nope, you are quite right. It should be set to `KeyStateEnabled`. ~copy~cut-pasta mistake.